src/main/java/org/zalando/nakadi/service/AuthorizationValidator.java

```diff
@@ -44,6 +44,21 @@ public void validateAuthorization(@Nullable final EventTypeAuthorization auth) t
         }
     }
 
+    public void validateWrite(final EventType eventType) {
+        if (eventType.getAuthorization() == null) {
+            return;
+        }
+        final MyEventTypeResource2 resource = new MyEventTypeResource2(
+                eventType.getName(), eventType.getAuthorization());
+        final boolean authorized = authorizationService.isAuthorized(
```

Comment:
isAuthorized throws PluginException, which could hold some technical information we should not expose to the users.